docs/iris/src/userguide/real_and_lazy_data.rst

```diff
@@ -59,10 +57,9 @@ When does my data become real?
 
```

Comment:
I only noticed this having read the built version of the docs, but I feel like we may need a small sentence here to introduce the section, maybe about most data being loaded lazy and then only becoming real when it's called, or something.  Probably not that, because it's not really correct, but it just feels as if this section launches straight into details without any proper reflection on the subtitle.  Alternatively, you could change the subtitle to reflect the section content better?